app/assets/styles/styles.css

```diff
@@ -1,3 +1,15 @@
+@font-face {
+  font-family: 'TrajanPro-Bold';
+  src: url('/../fonts/TrajanPro-Bold.eot?#iefix') format('embedded-opentype'),  url('/../fonts/TrajanPro-Bold.otf')  format('opentype'),
+       url('/../fonts/TrajanPro-Bold.woff') format('woff'), url('/../fonts/TrajanPro-Bold.ttf')  format('truetype'), url('/../fonts/TrajanPro-Bold.svg#TrajanPro-Bold') format('svg');
+}
```

Comment:
Getting close

- Aboslute links needs to be fixed. Remeber, just a `/`
- `.eot` font is a font for older IE browsers. Microst has done a good job deprecating those awful broswers. We don't need to include it anymore. Yay!
- In the day of modern broswers we can pretty safely get away with `woff` and `woff2`. See https://css-tricks.com/snippets/css/using-font-face/#article-header-id-1

Is there a woff2 font of TrajanPro?